app/src/main/res/xml/providers.xml

```diff
@@ -235,6 +235,19 @@
             port="587"
             starttls="true" />
     </provider>
+    <provider
+        name="Gandi"
+        domain="gandi.net"
```

Comment:
I don't think this is the domain name of the email addresses.
The dot should be escaped with a double back slash.

Reply:
This is true. Gandi does not support @gandi.net but uses the mail.gandi.net for custom domains. I might be wrong adding a provider that only supports custom domains.

Reply:
I have added Gandi, so at least people can select the right server settings.
I have also changed to port 465 SSL/TLS and fixed starttls="true" for IMAP